modules/RTC/JitsiLocalTrack.js

```diff
@@ -144,6 +158,35 @@ function JitsiLocalTrack(
 JitsiLocalTrack.prototype = Object.create(JitsiTrack.prototype);
 JitsiLocalTrack.prototype.constructor = JitsiLocalTrack;
 
+JitsiLocalTrack.prototype._addPeerConnection = function(tpc) {
+    if (this._isAttachedToPC(tpc)) {
```

Comment:
This `_isAttachedToPC(tpc)` one liner is used in TraceablePeerConnection. Should I remove the _ sign and make it public ? I was thinking that it's kind of library internal, but I'm still confused about this sing usage....